.gitignore

```diff
@@ -14,3 +14,4 @@ docs/reference/src/main/docbook/en-US/version_info.xml
 .forge_settings
 .factorypath
 atlassian-ide-plugin.xml
+bin/
```

Comment:
Isn't this entry ignoring https://github.com/forge/roaster/tree/master/dist/src/main/resources/bin?

Reply:
Yes, next commit will fix this